tools/c7n_tencentcloud/c7n_tencentcloud/query.py

```diff
@@ -226,50 +227,51 @@ def resources(self, params=None):
 
         if self.resource_manager.resource_type.paging_def:
             res = self.query_helper.paged_filter(self.resource_manager.config.region,
-                                            self.resource_manager.resource_type,
-                                            params)
+                                                 self.resource_manager.resource_type,
+                                                 params)
         else:
             res = self.query_helper.filter(self.resource_manager.config.region,
-                                       self.resource_manager.resource_type,
-                                       params)
+                                           self.resource_manager.resource_type,
+                                           params)
         self.augment(res)
         return res
 
     def get_permissions(self):
         return []
 
     def augment(self, resources):
-        for r in resources:
-            r["Tags"] = self.get_resource_tag(r)
-        return resources
+        return self.get_resource_tag(resources)
```

Comment:
its not clear this is needed in all cases, at least cvm are already returned with tags, which makes augmenting them for tags redundant.

Reply:
CVM list API is the only resource API which returns tags with the resource itself, as far as we know.

Reply:
sure, but we shouldn't fetch them again (extraneous api calls) for resources which already have tags present.